src/js/Search/SearchVideoWrapper.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-param-reassign */
+import { CLASSNAME, NUMBER, SNACKBAR_TEXT } from "../constants/index.js";
+import { messenger, MESSAGE } from "../messenger/index.js";
+import { renderSearchVideo, SEARCH_VIDEO_TEMPLATE } from "../Video/index.js";
+import { $, fetchYoutubeData, showModalSnackbar } from "../utils/index.js";
+
+export default class SearchVideoWrapper {
+  constructor() {
+    this.initializeVariables();
+    this.selectHTMLElements();
+    this.addMessageListeners();
+    this.addEventListeners();
+  }
+
+  initializeVariables() {
+    this.currentQuery = "";
+    this.currentNextPageToken = "";
+    this.videoItemsMap = new Map();
+  }
+
+  selectHTMLElements() {
+    this.$searchVideoWrapper = $(`.${CLASSNAME.SEARCH_VIDEO_WRAPPER}`);
+    this.$notFoundImg = $(`.${CLASSNAME.NOT_FOUND_IMAGE}`);
```

Comment:
```suggestion
this.$notFoundImge = $(`.${CLASSNAME.NOT_FOUND_IMAGE}`);
```

굳이 img로 줄여쓸 필요가 없는 것 같아요

Reply:
해당 부분은 리팩토링으로 인해 제거되었습니다.